### PR TITLE
chore(curriculum): Authors Page add quotes to strings

### PR DESCRIPTION
**Pull request by Supravisor**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Related to #51036

### CODE REVIEW

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/641da4b16937be43ba24c63d.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Chain another `.then()` to the existing `.then()` method. This time, pass in `da
 
 # --hints--
 
-You should use the `fetch()` method to make a `GET` request to `https://cdn.freecodecamp.org/curriculum/news-author-page/authors.json`.
+You should use the `fetch()` method to make a `GET` request to `"https://cdn.freecodecamp.org/curriculum/news-author-page/authors.json"`.
```
Comment (Sembauke) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/641da4b16937be43ba24c63d.md:17]:
```suggestion
You should use the `fetch()` method to make a `GET` request to `"https://cdn.freecodecamp.org/curriculum/news-author-page/authors.json"`.
```

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/641da42481d90c4314c99e94.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ Again, don't terminate the code with a semicolon yet.
 
 # --hints--
 
-You should use the `fetch()` method to make a `GET` request to `https://cdn.freecodecamp.org/curriculum/news-author-page/authors.json`.
+You should use the `fetch()` method to make a `GET` request to `'https://cdn.freecodecamp.org/curriculum/news-author-page/authors.json'`.
```
Comment (Sembauke) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/641da42481d90c4314c99e94.md:27]:
```suggestion
You should use the `fetch()` method to make a `GET` request to `"https://cdn.freecodecamp.org/curriculum/news-author-page/authors.json"`.
```

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/641da3c6b6fbd742bff6ee40.md [hunk 3]
```diff
@@ -15,11 +15,11 @@ Here is how you can make a `GET` request with the `fetch()` method:
 fetch("url-goes-here")
 ```
 
-Make a `GET` request to this URL: `https://cdn.freecodecamp.org/curriculum/news-author-page/authors.json`. Don't terminate your code with a semicolon yet.
+Make a `GET` request to this URL: `'https://cdn.freecodecamp.org/curriculum/news-author-page/authors.json'`. Don't terminate your code with a semicolon yet.
 
 # --hints--
 
-You should use the `fetch()` method to make a `GET` request to `https://cdn.freecodecamp.org/curriculum/news-author-page/authors.json`.
+You should use the `fetch()` method to make a `GET` request to `'https://cdn.freecodecamp.org/curriculum/news-author-page/authors.json'`.
```
Comment (Sembauke) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/641da3c6b6fbd742bff6ee40.md:22]:
```suggestion
You should use the `fetch()` method to make a `GET` request to `"https://cdn.freecodecamp.org/curriculum/news-author-page/authors.json"`.
```

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/641da51a9810e74411262fcc.md [hunk 4]
```diff
@@ -15,7 +15,7 @@ Chain `.catch()` to the last `.then()`. Pass in a callback function with `err` a
 
 # --hints--
 
-You should use the `fetch()` method to make a `GET` request to `https://cdn.freecodecamp.org/curriculum/news-author-page/authors.json`.
+You should use the `fetch()` method to make a `GET` request to `'https://cdn.freecodecamp.org/curriculum/news-author-page/authors.json'`.
```
Comment (Sembauke) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/641da51a9810e74411262fcc.md:18]:
```suggestion
You should use the `fetch()` method to make a `GET` request to `"https://cdn.freecodecamp.org/curriculum/news-author-page/authors.json"`.
```

**Review by Sembauke — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/641da3c6b6fbd742bff6ee40.md [hunk 5]
```diff
@@ -15,11 +15,11 @@ Here is how you can make a `GET` request with the `fetch()` method:
 fetch("url-goes-here")
 ```
 
-Make a `GET` request to this URL: `https://cdn.freecodecamp.org/curriculum/news-author-page/authors.json`. Don't terminate your code with a semicolon yet.
+Make a `GET` request to this URL: `'https://cdn.freecodecamp.org/curriculum/news-author-page/authors.json'`. Don't terminate your code with a semicolon yet.
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/641da3c6b6fbd742bff6ee40.md:18]:
```suggestion
Make a `GET` request to this URL: `"https://cdn.freecodecamp.org/curriculum/news-author-page/authors.json"`. Don't terminate your code with a semicolon yet.
```

**Review by naomi-lgbt — COMMENTED** [review]
Consistency nit.

**Review by huyenltnguyen — APPROVED** [review]

**Review by naomi-lgbt — APPROVED** [review]

**Review by Sembauke — APPROVED** [review]

**Pull request merged**